src/SonarScanner.MSBuild.Shim/SonarEngineWrapper.cs

```diff
@@ -44,6 +44,7 @@ public virtual bool Execute(AnalysisConfig config, string standardInput)
         var args = new ProcessRunnerArguments(javaExe, isBatchScript: false)
         {
             CmdLineArgs = javaParams.Any() ? [..javaParams, "-jar", engine] : ["-jar", engine],
+            WorkingDirectory = config.SonarScannerWorkingDirectory,
```

Comment:
Without this, the current working directory is not correct for the engine invocation and relatives paths are not resolved correctly (I think this is because we change the working directory of s4net.exe for some reason).

This line was copied from the scanner-cli invocation:
https://github.com/SonarSource/sonar-scanner-msbuild/blob/51d6d506df2d5d3310dbdf73962eec6529799276/src/SonarScanner.MSBuild.Shim/SonarScanner.Wrapper.cs?plain=1#L76C1-L76C68